wikichange/src/options/options.js

```diff
@@ -2,34 +2,42 @@
 
 /** Saves options to chrome.storage */
 function save_options() {
-    const color = document.getElementById('highlight-color').value;
-    chrome.storage.sync.set({
-        highlightColor: color
-    }, function () {
-        /** Update status to let user know options were saved. */
-        const status = document.getElementById('status');
-        status.textContent = 'Options saved.';
-        setTimeout(function () {
-            status.textContent = '';
-        }, 750);
-    });
+    const color = document.getElementById("highlight-color").value;
+    chrome.storage.sync.set(
+        {
+            highlightColor: color,
+        },
+        function () {
+            /** Update status to let user know options were saved. */
+            const status = document.getElementById("status");
+            status.textContent = "Options saved.";
+            setTimeout(function () {
+                status.textContent = "";
+            }, 750);
+        }
+    );
 }
 
 /** Restores select box and checkbox state using the preferences
  * stored in chrome.storage.
  */
 function restore_options() {
     /** Use default value highlightColor #AFE1AF */
-    chrome.storage.sync.get({
-        highlightColor: '#AFE1AF'
-    }, function (items) {
-        console.log(items);
-        document.getElementById('highlight-color').value = items.highlightColor;
-    });
+    chrome.storage.sync.get(
+        {
+            highlightColor: "#AFE1AF",
+        },
+        function (items) {
+            document.getElementById("highlight-color").value = items.highlightColor;
+        }
+    );
```

Comment:
Not sure what this is doing, but the color never changes back to the default highlight for me. @RichartE